Configurateur/front/lib/screens/login/login.dart

```diff
@@ -26,175 +29,208 @@ class LoginScreen extends StatefulWidget {
+                                            }
+                                        });
+                              }
+                            },
+                            style: ElevatedButton.styleFrom(
+                              shape: RoundedRectangleBorder(
+                                borderRadius: BorderRadius.circular(30.0),
+                              ),
                             ),
-                          ),
-                          child: Text(
-                            "Se connecter",
-                            style: TextStyle(
-                              fontSize: 18,
-                              color: Provider.of<ThemeService>(context).isDark
-                                  ? darkTheme.primaryColor
-                                  : lightTheme.primaryColor,
+                            child: Text(
+                              "Se connecter",
+                              style: TextStyle(
+                                color: Provider.of<ThemeService>(context).isDark
+                                    ? darkTheme.primaryColor
+                                    : lightTheme.primaryColor,
+                                fontSize: 18,
+                              ),
                             ),
                           ),
                         ),
-                      ),
-                      InkWell(
-                        key: const Key('register'),
-                        onTap: () {
-                          context.go("/register");
-                        },
-                        child: Padding(
-                          padding: const EdgeInsets.all(10.0),
-                          child: Row(
-                              mainAxisAlignment: MainAxisAlignment.center,
-                              children: const <Widget>[
-                                Text("Nouveau sur la plateforme ? "),
-                                Text(
-                                  'Créer un compte.',
-                                  style: TextStyle(color: Colors.blue),
-                                ),
-                              ]),
+                        InkWell(
+                          key: const Key('register'),
+                          onTap: () {
+                            context.go("/register");
+                          },
+                          child: const Padding(
+                            padding: EdgeInsets.all(10.0),
+                            child: Row(
+                                mainAxisAlignment: MainAxisAlignment.center,
+                                children: <Widget>[
+                                  Text("Nouveau sur la plateforme ? "),
+                                  Text(
+                                    'Créer un compte.',
+                                    style: TextStyle(color: Colors.blue),
+                                  ),
+                                ]),
```

Comment:
Maybe add , after the ] so that the file is more readable

Reply:
done